### ISSUE TITLE
Should we default to disabling a rule?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/goodparts#281]
Each time the `eslint` package adds a rule `goodparts` CI "_fails_" _unnecessarily_... 
e.g: https://travis-ci.org/dwyl/goodparts/builds/196241118#L698

_could/should_ we default to _disabling_ a rule if it's "_not configured_" to avoid this...?
I was trying to add a `CONTRIBUTING.md` file in: https://github.com/dwyl/goodparts/pull/280
but could not because the build fails... 😞 

_thoughts_?

**Comment by Charlotteis** [comment]
I would pass the test suite if a rule is not configured. I'd still want to know there was an error, but I wouldn't want to fail the build.

**Comment by shouston3** [comment]
Couldn't we just change the `package.json` eslint version from

```
"eslint": "^3.9.1",
```

to

```
"eslint": "~3.9.1",
```

And then let [`david-dm`](https://david-dm.org/) notify us of new [eslint releases](https://github.com/eslint/eslint/releases)

Would that work?

**Comment by Charlotteis** [comment]
I think greenkeeper could let us know of new releases, by PRing updates to node packages which we could address?

**Comment by nelsonic** [comment]
Does Greenkeeper still require "**admin access**" to all repos in the Org to do it's thing? 🤔 
see: https://github.com/dwyl/hq/issues/100 😕

**Comment by Charlotteis** [comment]
@nelsonic https://github.com/integration/greenkeeper :)

**Comment by nelsonic** [comment]
@Charlotteis thanks for updating our knowledge on Greenkeeper. 👍 
![Greenkeeper Permissions](https://cloud.githubusercontent.com/assets/194400/22767625/9df8bdc4-ee74-11e6-9172-74309607beea.png)
(_no longer **admin** access..._)

**Comment by Fl4m3Ph03n1x** [comment]
I was moved here via thread https://github.com/dwyl/goodparts/issues/272#issuecomment-310834719

I see this issues was closed, is any assistance still needed?

**Comment by nelsonic** [comment]
@Fl4m3Ph03n1x yes, we need to `default` to ignoring new rules. if you have time to implement, please go for it! 👍

**Comment by Fl4m3Ph03n1x** [comment]
@nelsonic I tried to find the time, but unfortunately I just can't dedicate it. I apologize for not being more useful, but at least I will try to help on what I can.

**Comment by nelsonic** [comment]
@Fl4m3Ph03n1x _never_ apologise for not having "enough time" that's the human condition. ⌛️ 
you've made great contributions _already_ and we're super grateful! 🎉
